tpm.c

```diff
@@ -137,16 +102,14 @@ int tpm_extend_pcr(struct tpm *t, u32 pcr, u16 algo,
 	int ret = 0;
 
 	if (t->buff == NULL) {
-		ret = -EINVAL;
-		goto out;
+		return -EINVAL;
 	}
 
 	if (t->family == TPM12) {
 		struct tpm_digest d;
 
 		if (algo != TPM_ALG_SHA1) {
-			ret = -EINVAL;
-			goto out;
+			return -EINVAL;
```

Comment:
Ditto on brackets anywhere the if just became a one line return statement.

Reply:
ack